js/hw_22.js

```diff
@@ -0,0 +1,74 @@
+// 1
+const visitorName = prompt(
+  "Welcome, traveler from distant land. Please, enter your name:"
+);
+alert("Hello, " + visitorName + "!");
+
+// 2
+const currentYear = 2021;
+const birthYear = prompt("What year where you born in?");
+const visitorAge = currentYear - birthYear;
+alert(`So, you are actually ${visitorAge} years old...`);
+
+// 3
+const squareSide = +prompt(
+  "Want to know the perimetr? Enter the length of the side of your square:"
+);
+alert(`The perimetr of your square is ${squareSide * 4}`);
+
+// 4
+const PI = 3.14159;
+const circleRadius = +prompt("Enter the circle radius:");
+alert(`The area of your circle is ${PI * circleRadius ** 2}`);
+
+// 5
+const distance = +prompt("How far (in km) is the town you are going to visit?");
+const travelTime = +prompt(
+  "How long (in hours) do you want your trip to last?"
+);
+alert(
+  `You will need to move with an avarage speed of ${
+    distance / travelTime
+  } km / hour`
+);
+
+// 6
+const visitorAmount = +prompt(
+  "This is a converter from dollars to euro. Enter the amount of $ you want to convert:"
+);
+const exchangeRateEuro = 0.83;
+alert(`$${visitorAmount} = ${exchangeRateEuro * visitorAmount} euro`);
+
+// 7
+const fleshSize = +prompt("How big is your flesh drive (in GB)?");
+const fleshSizeMb = fleshSize * 1024;
+const fileSize = 820;
+const amountLeft = fleshSizeMb % fileSize;
+alert(
+  `Your flesh drive can contain ${
+    (fleshSizeMb - amountLeft) / 820
+  } file(s) of 820MB each`
+);
+
+// 8
+const cashInWallet = +prompt("How much money do you have?");
+const bookCost = +prompt("How much does the book cost?");
+const cashLeft = cashInWallet % bookCost;
+const bookAmount = (cashInWallet - cashLeft) / bookCost;
+alert(
+  `Congrats! You can by ${bookAmount} copy(-ies) and still have $${cashLeft} left`
+);
+
+// 9
+const numThreeDigit = +prompt("Please, enter a 3 digits number:");
+const numLast = numThreeDigit % 10;
+const numSecond = ((numThreeDigit % 100) - (numThreeDigit % 10)) / 10;
```

Comment:
можна реюзати змінні, щоб менше писати
```suggestion
const numSecond = ((numThreeDigit % 100) - numLast) / 10;
```